layers/meta-balena-jetson/recipes-bsp/tools/files/mark-active-slot.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Mark boot slot as successful once the system is up
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh -c /usr/bin/mark_active_tegra_boot_slot.sh
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
@acostach is this service running every single boot, even if no HUP happened?

Reply:
Hi @alexgg , indeed, it needs to run every boot, otherwise the tegra bootloaders will consider that the current one failed and decrease the current slot's priority, causing the alternate to be loaded once the retries number is achieved.

Disabling redundant boot after marking a boot sequence as successful unfortunately triggers the default slot _a to be loaded, so the only option for now is to mark every boot if we want to have the kernel and dtb loaded from the selected raw partitions, _a or _b.